src/Workspaces/Core/Portable/Shared/Extensions/SyntaxNodeExtensions.cs

```diff
@@ -762,8 +762,7 @@ public static SyntaxNode GetParent(this SyntaxNode node, bool ascendOutOfTrivia)
             var parent = node.Parent;
             if (parent == null && ascendOutOfTrivia)
             {
-                var structuredTrivia = node as IStructuredTriviaSyntax;
-                if (structuredTrivia != null)
```

Comment:
📝  This file should not be part of this change.

Reply:
Why not?

Reply:
It is not related to the subject matter of this pull request, therefore unnecessarily increasing the complexity and time for review, and producing misleading git history results.